packages/systemd/systemd.spec

```diff
@@ -11,7 +11,8 @@ Source0: https://github.com/systemd/systemd/archive/v%{version}/systemd-%{versio
 Source1: run-tmpfiles.conf
 Source2: systemd-modules-load.conf
 
-# Local changes.
+# Local patches that can be dropped when we have an SELinux policy that
+# limits access to system files on the data volume.
 Patch9001: 9001-move-stateful-paths-to-ephemeral-storage.patch
 Patch9002: 9002-do-not-create-unused-state-directories.patch
 Patch9003: 9003-use-absolute-path-for-var-run-symlink.patch
```

Comment:
I'm not sure I understand how this comment about SELinux relates to these patches - why would we start to create unused state directories, and why would we not longer want an absolute path for a /var/run symlink?  I'm not sure there's enough context.

Reply:
I'll add a separate description for 9003, since yeah that one isn't affected by SELinux.

Aside from the coredump handler directory, the other unused directories are `/var/{lib,log,cache}/private` which are the boundary directories for the [runtime directories](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#RuntimeDirectory=) that systemd can create for services.

Those will go back to their stock behavior after the SELinux policy changes.